CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to CRA Template Truemark
+
+Thank you for your interest in contributing to our project! We appreciate your interest in contributing.
+
+## How to Contribute
+
+- Fork the Template: On GitHub, click "Fork" to create your own copy.
+- Clone the forked repository to your local machine.
+
+    ```md 
+    git clone https://github.com/TruemarkDev/cra-template-truemark.git for https.
+
+    git clone git@github.com:TruemarkDev/cra-template-truemark.git for ssh.
+    ```
+
+- Install dependencies using `npm install`.
+- Start the project using `npm start`.
+- Check out to a new branch.
+- Make your changes and commit them.
+- Push your changes to repository.
+- Create a pull request with a clear title and description.
+
+## Code Guidelines
+
+- Follow consistent coding style.
+- Write clear and meaningful commit messages.
+
+## Update documentation for the changes made if needed
+
+If your changes affect the functionality of the library or introduce new features, please update the documentation to reflect these changes. This includes:
+
+-  Adding new library or updating the packages.
+-  Adding any new configuration options or usage instructions.
+
+## Creating a Pull Merge
+
+You need to create a PR properly. While creating a PR, you need to add the description of the tasks or features done. The PR must be marked as draft if all the task has not been completed and the remaining tasks needs to be mentioned in Remaining Tasks section as follows. Also, you can add the UI of the part of work done.
+
+```md
+
+Tasks Completed
+
+// Completed tasks
+
+Remaining Tasks
+
+// Remaining tasks if any
+
+Steps To Test Features
+
+// Steps that needs to be followed to test the feature
+
+Screenshots/Video
+
+// Media to show the UI of the task you performed
+
+```
+
+## Creating an issue from a repository
+
+- On GitHub.com, navigate to the main page of the repository.
+- Under your repository name, click Issues.
+- Click New issue.
+- In the "Title" field, type a title for your issue.
+- In the comment body field, type a description of your issue.
+- When you're finished, click Submit new issue.
+
+This is how you can contribute to cra-template-truemark.
```

Comment:
Previous documentation was correct one. we need to only add that we need to create issue while adding fixes for the template.